### PR TITLE
feat: wrapper types tests 🧪

### LINKED ISSUE #297: feat(test): wrapper primitives tests 🧪
# Feature Request

**Describe the Feature Request**

In `primitives` folder, we have a lot of types that end with `Wrapper`, they are wrappers around `starknet_api` types such that we can use them within substrate's runtime.

**Acceptance Criteria**

- PR must increase coverage of the repo
- All `Wrapper` types should have according unit tests for conversion

You can start by looking for `to_starknet_call_entry_point`, `to_starknet_contract_class`, `to_starknet` methods and replace them with `TryInto` trait implementation.

Then you can look for all implementations of the `TryInto` trait and write according unit tests.

### PR DESCRIPTION
**Pull request by EvolveArt**
# Pull Request type

Please check the type of change your PR introduces:

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Build-related changes
- [ ] Documentation content changes
- [x] Testing
- [ ] Other (please describe):

# What is the current behavior?

Issue Number: Closes #297

### CODE REVIEW

File: crates/primitives/starknet/src/execution/call_entrypoint_wrapper.rs [hunk 1]
```diff
@@ -63,14 +64,13 @@ impl CallEntryPointWrapper {
     }
 
     /// Convert to Starknet CallEntryPoint
-    pub fn to_starknet_call_entry_point(&self) -> CallEntryPoint {
+    fn to_starknet_call_entry_point(&self) -> Result<CallEntryPoint, StarknetApiError> {
```
Comment (LucasLvy) [inline crates/primitives/starknet/src/execution/call_entrypoint_wrapper.rs:67]: impl tryfrom trait

File: crates/primitives/starknet/src/execution/call_entrypoint_wrapper.rs [hunk 2]
```diff
@@ -127,12 +130,21 @@ impl CallEntryPointWrapper {
 impl Default for CallEntryPointWrapper {
     fn default() -> Self {
         Self {
-            class_hash: Some(ClassHashWrapper::default()),
+            class_hash: None,
             entrypoint_type: EntryPointTypeWrapper::External,
             entrypoint_selector: Some(H256::default()),
-            calldata: BoundedVec::try_from(vec![U256::zero(); 32]).unwrap(),
+            calldata: BoundedVec::try_from(vec![]).unwrap(),
```
Comment (LucasLvy) [inline crates/primitives/starknet/src/execution/call_entrypoint_wrapper.rs:136]: use default ?

**Review by LucasLvy — CHANGES_REQUESTED** [review]

File: crates/primitives/starknet/src/execution/call_entrypoint_wrapper.rs (hunk 1 above)
Comment (EvolveArt) [reply crates/primitives/starknet/src/execution/call_entrypoint_wrapper.rs:67]: there is already try_into this is just a helper internal function

**Review by LucasLvy — CHANGES_REQUESTED** [review]

**Review by LucasLvy — APPROVED** [review]
lgtm

**Pull request merged**